app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -263,6 +264,7 @@ def handle_expired_status_update(enrollment, response, response_message)
       passed: false,
       reason: 'Enrollment has expired',
       job_name: self.class.name,
+      tmx_status: enrollment&.profile&.fraud_pending_reason || :threatmetrix_pass,
```

Comment:
I'm not familiar with this pattern of `|| :threatmetrix_pass`. If the `fraud_pending_reason` isn't passed in, what does that symbol (:threatmetrix_pass) resolve to? Does the symbol get turned into a string in the log message?

Reply:
There's an enum in `profile` that has only 2 statuses, review & reject. S if `fraud_pending_reason` is not either one of those, it is `nil` and corresponds to a pass. So in that code, it'll bypass the first part of the `||` logical operator and proceed to the second as the value, `:threatmetrix_pass`. A symbol when serialized to JSON will show up as a string, as seen in the screenshots above. 

We seem to use both symbols and strings in the codebase, and I'm open to whatever is more consistent. Symbols are usually more efficient though, I think. And in this case it's just for JSON logging.

Regarding pass = `nil`, going to add a quote from Matt in Slack to elaborate because he said it best:
>I think the issue is that we don't bother logging that they passed ThreatMetrix, and the field was thus called fraud_pending_reason. I wonder if it's safe to assume nil == 'passed'